rwengine/src/objects/VehicleObject.cpp

```diff
@@ -385,6 +385,18 @@ void VehicleObject::tickPhysics(float dt) {
             brakeF = 2.f * std::min(1.f + kM, 4.f);
         }
 
+        if (isStopped() && std::abs(throttle) < 0.1f) {
+            btVector3 v = collision->getBulletBody()->getLinearVelocity();
+            v.setX(0.f);
```

Comment:
I think you also want to set the y axis to zero, or else the vehicle is possible to drift in one direction.

Reply:
Good point, thanks!

Reply:
Glad I could help :+1: